uint8 MODE_EFFORT = 1
uint8 MODE_VELOCITY = 2
uint8 MODE_POSITION = 3

uint8 mode
float32 cmd